cocos/2d/assembler/sprite/simple.ts

```diff
@@ -170,6 +142,42 @@ export const simple: IAssembler = {
             vData[27] = ar + cttx;
             vData[28] = br + dtty;
         }
+    },
+
+    fillBuffers (sprite: Sprite, renderer: Batcher2D) {
+        if (sprite === null) {
+            return;
+        }
+
+        const vData = sprite.renderData!.vData!;
+        if (sprite.node.hasChangedFlags || (this.uiTransCache && this.uiTransCache._rectDirty)) {
+            this.updateWorldVerts(sprite, vData);
+            this.uiTransCache._rectDirty = false;
+        }
```

Comment:
```
if (sprite.node.hasChangedFlags) {
    this.updateWorldVerts(sprite, vData);
}
```

Reply:
will be update twice，node.hasChangedFlags still true until frame finish